app/redux/sagas/VelocityDropSaga.js

```diff
@@ -0,0 +1,28 @@
+import { takeLatest, select, put, call, all } from 'redux-saga/effects';
+import { 
+    ADD_REP_DATA, 
+} from 'app/configs+constants/ActionTypes';
+import * as SetsSelectors from 'app/redux/selectors/SetsSelectors';
+import * as RepDataMap from 'app/utility/RepDataMap';
+import * as VelocityDropActionCreators from 'app/redux/shared_actions/VelocityDropActionCreators';
+import * as SettingsSelectors from 'app/redux/selectors/SettingsSelectors';
+
+const VelocityDropSaga = function * VelocityDropSaga() {
+    yield takeLatest(ADD_REP_DATA, playVelocityDropAudio);
+};
+
+function* playVelocityDropAudio(action) {
+    const velocityThreshold = yield select(SettingsSelectors.getVelocityDropThreshold);
+    const workingSet = yield select(SetsSelectors.getWorkingSet);
+    const workingSetReps = workingSet.reps
+    const lastRep = workingSetReps[workingSetReps.length - 2];
```

Comment:
I think you’re still doing a drop instead of a velocity target / threshold.

Point is, is the rep you JUST DID too slow? If so, beep. That's it.

Reply:
I'm comparing the rep you just did against the last. Are we instead choosing to see if it is below a certain velocity in general? e.g if I set .3 in settings, would ANY rep below .3 issue a beep?